CHANGELOG/CHANGELOG-1.5.1.md

```diff
@@ -0,0 +1,16 @@
+# v1.5.1 
+
+## Changelog since v1.5.0 
+
+## Changes by Kind 
+
+### Features 
+
+- CSM 1.5.1 release specific changes. ([#582](https://github.com/dell/csm/issues/582))
+- Show volumes associated with the tenant from the k8s server. ([#408](https://github.com/dell/csm/issues/408))
+- CSI PowerScale documentation misses the privileges to user to use CSM replication. ([#516](https://github.com/dell/csm/issues/516))
```

Comment:
There was no fix required for https://github.com/dell/csm/issues/516. The documentation has everything and I closed the issue in 1.5.0 timeframe. Can I understand why the milestone is now 1.5.1? Am I missing anything?

Reply:
@harshaatdell the milestone is still tied to the GitHub issue. Since no work was delivered for 516, please remove the milestone and it will be removed from the change-log on the next iteration.

Reply:
> There was no fix required for https://github.com/dell/csm/issues/516. The documentation has everything and I closed the issue in 1.5.0 timeframe. Can I understand why the milestone is now 1.5.1? Am I missing anything?

Reply:
> There was no fix required for https://github.com/dell/csm/issues/516. The documentation has everything and I closed the issue in 1.5.0 timeframe. Can I understand why the milestone is now 1.5.1? Am I missing anything?

Reply:
> There was no fix required for https://github.com/dell/csm/issues/516. The documentation has everything and I closed the issue in 1.5.0 timeframe. Can I understand why the milestone is now 1.5.1? Am I missing anything?

Reply:
> There was no fix required for https://github.com/dell/csm/issues/516. The documentation has everything and I closed the issue in 1.5.0 timeframe. Can I understand why the milestone is now 1.5.1? Am I missing anything?

Reply:
> @harshaatdell the milestone is still tied to the GitHub issue. Since no work was delivered for 516, please remove the milestone and it will be removed from the change-log on the next iteration.

Reply:
> > There was no fix required for https://github.com/dell/csm/issues/516. The documentation has everything and I closed the issue in 1.5.0 timeframe. Can I understand why the milestone is now 1.5.1? Am I missing anything? 
> 
>

Reply:
> > > There was no fix required for https://github.com/dell/csm/issues/516. The documentation has everything and I closed the issue in 1.5.0 timeframe. Can I understand why the milestone is now 1.5.1? Am I missing anything? 
> > 
> > 
> 
>

Reply:
> There was no fix required for https://github.com/dell/csm/issues/516. The documentation has everything and I closed the issue in 1.5.0 timeframe. Can I understand why the milestone is now 1.5.1? Am I missing anything?

Reply:
> > There was no fix required for https://github.com/dell/csm/issues/516. The documentation has everything and I closed the issue in 1.5.0 timeframe. Can I understand why the milestone is now 1.5.1? Am I missing anything? 
> 
>

Reply:
> > > There was no fix required for https://github.com/dell/csm/issues/516. The documentation has everything and I closed the issue in 1.5.0 timeframe. Can I understand why the milestone is now 1.5.1? Am I missing anything? 
> > 
> > 
> 
>

Reply:
https://github.com/users/ioeicicdsvc/achievements/pair-extraordinaire

Reply:
> https://github.com/users/ioeicicdsvc/achievements/pair-extraordinaire